sdk/src/types/block/output/account.rs

```diff
@@ -87,9 +87,9 @@ impl AccountOutputBuilder {
     }
 
     /// Creates an [`AccountOutputBuilder`] with a provided rent structure.
-    /// The amount will be set to the minimum storage deposit.
-    pub fn new_with_minimum_storage_deposit(rent_structure: RentStructure, account_id: AccountId) -> Self {
-        Self::new(OutputBuilderAmount::MinimumStorageDeposit(rent_structure), account_id)
+    /// The amount will be set to the rent cost of the resulting output.
+    pub fn new_with_minimum_amount(rent_structure: RentStructure, account_id: AccountId) -> Self {
```

Comment:
I think I would rename these to either `new_with_rent_cost` or `new_with_rent_cost_amount` to better match the variant name

Reply:
?

Reply:
I don't like those names really :/